lib/server/sthServer.js

```diff
@@ -63,6 +63,10 @@ function doStartServer(host, port, callback) {
             reply({ error: 'BadRequest', description: error.output.payload.message }).code(400);
         }
 
+        const list = ['min', 'max', 'sum', 'sum2', 'occur', 'all'];
+        const joinedList = '(' + list.join('|') + ')';
+        const regex = new RegExp('^' + joinedList + '(,' + joinedList + ')*$');
```

Comment:
Please use more meaningful names names for the variables. For instance:

```suggestion
        const aggList = ['min', 'max', 'sum', 'sum2', 'occur', 'all'];
        const joinedAggList = '(' + aggList.join('|') + ')';
        const aggRegex = new RegExp('^' + joinedAggList + '(,' + joinedAggList + ')*$');
```

Reply:
Fixed in [3b51b4c](https://github.com/telefonicaid/fiware-sth-comet/pull/605/commits/3b51b4c42e14b77f331525db1fcd0926a79afffd)